modules/tinymce/src/plugins/lists/main/ts/core/Delete.ts

```diff
@@ -273,7 +273,18 @@ const hasListSelection = (editor: Editor) => {
 const backspaceDeleteRange = (editor: Editor): boolean => {
   if (hasListSelection(editor)) {
     editor.undoManager.transact(() => {
+      // Some delete actions may prevent the input event from being fired. If we do not detect it, we fire it ourselves.
+      let fireInput = true;
```

Comment:
could it make sense to change this in something like: `shouldFireInput`?

Reply:
probably a good idea.